stdlib/universe/window_test.go

```diff
@@ -72,24 +74,24 @@ func TestWindowOperation_Marshaling(t *testing.T) {
 	querytest.OperationMarshalingTestHelper(t, data, op)
 }
 
-func TestFixedWindow_PassThrough(t *testing.T) {
-	executetest.TransformationPassThroughTestHelper(t, func(d execute.Dataset, c execute.TableBuilderCache) execute.Transformation {
-		fw := universe.NewFixedWindowTransformation(
-			d,
-			c,
-			execute.Bounds{},
-			execute.Window{
-				Every:  values.ConvertDuration(time.Minute),
-				Period: values.ConvertDuration(time.Minute),
-			},
-			execute.DefaultTimeColLabel,
-			execute.DefaultStartColLabel,
-			execute.DefaultStopColLabel,
-			false,
-		)
-		return fw
-	})
-}
+// func TestFixedWindow_PassThrough(t *testing.T) {
```

Comment:
Why was this test removed?

Reply:
I think it was commented out for a compilation error and I wasn't sure what it did. I'll see if I can readd it.